source/GameData.cpp

```diff
@@ -231,16 +249,8 @@ void GameData::LoadShaders()
 
 double GameData::GetProgress()
 {
-	// Cache progress completion seen, so clients are
-	// isolated from the loading implementation details.
-	static bool initiallyLoaded = false;
-	if(initiallyLoaded)
-		return 1.;
-
-	double val = min(min(spriteQueue.GetProgress(), Audio::GetProgress()), objects.GetProgress());
-	if(val >= 1.)
-		initiallyLoaded = true;
-	return val;
+	double spriteProgress = static_cast<double>(spriteLoadingProgress) / totalSprites;
+	return min(min(spriteProgress, Audio::GetProgress()), objects.GetProgress());
```

Comment:
```suggestion
	return min(spriteProgress, Audio::GetProgress(), objects.GetProgress());
```
would this work too?
as min accepts a std initializer_list
just a question

Reply:
It would have to be `min({one, two, three})` but yes thanks!